### ISSUE TITLE
Adding campaign end date to the Campaigns sheet that gets downloaded

### ISSUE THREAD

**Issue by Payoshni** [PrathamBooks/frp#289]
@kuchlous 
Asking for this addition otherwise finding out closed campaigns becomes a nightmare as the data sheet is expanding.

If there is end date filter (actual end date that too), I can filter campaigns to the latest date and see which ones have closed to start sending out books.

**Comment by kuchlous** [comment]
Do you want a filter or just an extra column? With extra column, you can
always filter in Excel.

**Comment by Payoshni** [comment]
extra column would be nice.

Regards,
Payoshni

“Someone reading a Book is a sign of order in the world”

-Mary Ruefle

**Comment by kuchlous** [comment]
Start date is already there. Do you need an additional end date? Should I
just replace the start date with End date? You also have days left in the
sheet.

Adding a new column is simple enough, just do not want to add redundant
info and make the sheet bloated.

**Comment by Payoshni** [comment]
start date is actually of no use to me.

Regards,
Payoshni

“Someone reading a Book is a sign of order in the world”

-Mary Ruefle

**Comment by kuchlous** [comment]
OK, will replace it then.

**Comment by Payoshni** [comment]
yes. will work. thanks

Regards,
Payoshni

“Someone reading a Book is a sign of order in the world”

-Mary Ruefle

**Comment by kuchlous** [comment]
Fixed.

**Comment by Payoshni** [comment]
Hi @kuchlous  This solution is not helping because the closing date on the campaign sheet shows the 'proposed' closing date which is calculated by the system as against the 'actual' closing date. So it doesn't make the filtering easier at all.

e.g: a campaign's original closing date is 24th sept. It got fully funded on 18th september.  When i download the sheet, ti shows 24thg as againt 18th, which means when I filter to see on 20th september which campaigns have funded so that i can start sending books, this campaign doesn't show up as closed with a date filter.

Case in point: https://donateabook.org.in/campaign/91 which closed a week or so ago but the end date shows 18th oct.

**Comment by kuchlous** [comment]
@Payoshni , I understand. Currently we are not maintaining the date of the actual close. I can add that but will take some time.

In the meanwhile we also print the status of the campaign, shouldn't that be helpful?

**Comment by Payoshni** [comment]
Yes, right now tracking it with Status. The thing is, now that the
campaigns are increasing in numbers, when i choose 'Closed' also and apply
another filter for 100% & more, there are still MANY campaigns. So each
time i have to cross check this sheet vs the other sheet we are maintaining
for stocks and it's a nightmare, especially with so many Teach For India
campaigns coming up where org name is the same :(

This can wait, let;s add the actual closing date. No other solution will
ease this process, unless we work on an automated solution when every
closed campaigns automatically goes into a separate sheet which is all
online :) Then this manual pain reduces.

Regards,
Payoshni

“Someone reading a Book is a sign of order in the world”

-Mary Ruefle
